src/main/java/fr/rader/boblite/Main.java

```diff
@@ -23,242 +29,123 @@ public void start() {
-
-                // check if the packet id is a chat message packet,
-                // and if the removeChat checkbox is checked
-                if (menu.isRemoveChatChecked() && packetID == chatPacketID) {
-                    // if bob has to remove the chat packet,
-                    // we skip it, and we don't write it
-                    reader.skip(size - 1);
-                    continue;
-                }
-
-                // check if the packet id is a time packet,
-                // and if the changeTime checkbox is checked
-                if (menu.isChangeTimeChecked() && packetID == timePacketID) {
-                    // writing packet header
-                    writer.writeInt(timestamp);
-                    writer.writeInt(size);
-                    writer.writeVarInt(packetID);
-
-                    // writing world age
-                    writer.writeLong(reader.readLong());
-                    // writing new time
-                    writer.writeLong(menu.getNewTime());
-
-                    // skip the next 8 bytes (old world time)
-                    reader.skip(8);
-
-                    continue;
-                }
-
-                // check if the packet id is a weather packet,
-                // and if the removeRain checkbox is checked
-                if (menu.isRemoveRainChecked() && packetID == weatherPacketID) {
-                    // get the reason, we don't want to ignore it because
-                    // this packet isn't always used for the weather
-                    int reason = reader.readByte();
-
-                    // reason == 1 -> end raining
-                    // reason == 2 -> begin raining
-                    // reason == 7 -> rain level change
-                    // reason == 8 -> thunder level change
-                    // iirc reason 1 and 2 are swapped
-                    if (reason != 1 && reason != 2 && reason != 7 && reason != 8) {
-                        // if the packet does not change the rain, we write it
-                        // packet header
-                        writer.writeInt(timestamp);
-                        writer.writeInt(size);
-                        writer.writeVarInt(packetID);
-
-                        // packet content
-                        writer.writeByte(reason);
-                        writer.writeFloat(reader.readFloat());
-                    } else {
-                        // we skip the next 4 bytes (new rain/thunder level)
-                        reader.skip(4);
-                    }
+        if (tempFileDirectory == null) {
+            System.exit(0);
+        }
 
-                    continue;
-                }
+        // Okay, time to get working!
+
+        // Get the total number of system CPU threads.
+        int totalSystemThreadNumber = Runtime.getRuntime().availableProcessors();
+        int executorThreadNumber;
+        if (replays.size() < totalSystemThreadNumber){
```

Comment:
Add a space before the `{`

Reply:
Resolved in commit [fix code formatting - 63d35e3](https://github.com/RaderRMT/BobLite/pull/1/commits/63d35e385e87abea775c880b09dfc2494ea08a84)